src/MicrosoftTeams.ts

```diff
@@ -330,12 +331,9 @@ namespace microsoftTeams
             websiteUrl?: string;
 
             /**
```

Comment:
Yup, it's being replaced with entity id. I know they're not entirely the same thing, but they serve the same basic intent. Entity Id is actually slightly more powerful since it'll be exposed to the content frame and not just the settings frame.